### ISSUE TITLE
Iframe causing major offset in mobile view

### ISSUE THREAD

**Issue by scharmetant** [fancyapps/fancybox#1913]
When opening an AJAX content with fancybox, it the returned content contains an iframe, the fancybox window somehow gets pushed down so much that it is not visible, unless you scroll an entire screen to reach it.

To reproduce the problem: 

1. Create a page with an iframe inside (for example an embed from slides.com)
2. Call this page in Ajax, using fancybox
3. View in a mobile, or using a mobile simulator : you will see that the fancybox window is not visible unless you scroll down

The only way to avoid the problem, is to manually set a small size to the iframe, 
for ex: width="300" height="300"

**Comment by scharmetant** [comment]
The iframe must not be scrollable in this case, I use an embed from slides.com.
On way to fix the issue is fix the width to avoid the scrolling : 

 <iframe  src="//slides.com/artmajeur/selling_original_en/embed?style=light" width="300" height="300" scrolling="no" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>


The problem is also solved when, **as you suggest**, the iframe is made scrollable, wich seems the best solution : 
  <div  class="embed-responsive embed-responsive-4by3">
                <iframe class="embed-responsive-item" [...]


Good job !

**Comment by fancyapps** [comment]
I checked what code slide.com produces and it is just as I suspected. They have `width="576"` and therefore it breaks vertical centering on smaller screens. You see, fancybox uses "ghost" element (see https://css-tricks.com/centering-in-the-unknown/) and this seems to be the best option with the only drawback - if elements are too wide, they are pushed below "ghost" element. Saldy, it is 2018 and there are still no better solution, flexbox and grid has their flaws, too.
